luatest/helpers.lua

```diff
@@ -69,8 +69,7 @@ function helpers.retrying(config, fn, ...)
         if (clock.time() - started_at) > timeout then
             return fn(...)
         end
-        log.debug('Retrying in ' .. delay .. ' sec. due to error:')
-        log.debug(result)
+        log.debug('Retrying in %d sec. due to error: %s', delay, result)
```

Comment:
I am not sure that it should be `debug`. Looks like `verbose` rather than `debug`.

Reply:
I've fixed that.